docs/source/logging/logging.md

```diff
@@ -45,4 +51,44 @@ To start this example execute:
 RUST_LOG=info cargo run
 ```
 
-The full [example](https://github.com/scylladb/scylla-rust-driver/tree/main/examples/logging.rs) is available in the `examples` folder
\ No newline at end of file
+The full [example](https://github.com/scylladb/scylla-rust-driver/tree/main/examples/logging.rs) is available in the `examples` folder.
+You can run it from main folder of driver repository using `RUST_LOG=trace SCYLLA_URI=<scylla_ip>:9042 cargo run --example logging`.
+
+## Using log
+
+To collect tracing events using log collector you first need to enable `log` feature on `tracing` crate.
+You can use `cargo add tracing -F log` or edit `Cargo.toml`:
+```toml
+tracing = { version = "0.1.40" , features = ["log"] }
+```
+then you can setup `env_logger` os some other logger and it will output logs from the driver:
```

Comment:
Will enabling `log` feature in the end crate's Cargo.toml suffice, or is it required to have it enabled for scylla crate as well?
Another question. What if the end user crate uses `tracing` crate in version that is incompatible with the version that `scylla` crate uses?

Reply:
1. Yes, it is enough
2. Not sure what you are asking. Are you asking if adding hypotetical tracing 0.2 to end user Cargo.toml and enabling log feature there will enable log feature in our tracing 0.1? No, it won't

Reply:
Quoting comment from the tracing issue I opened (https://github.com/tokio-rs/tracing/issues/2913#issuecomment-2007815042 ):
> A library should not generally enable the log or log-always feature flags, and generally should never be responsible for setting the default subscriber. Instead, user code that depends on the library should enable tracing's log crate support if it wants to, and set the default subscriber if it wishes to use tracing. I don't think the calls to with_current_subscriber are necessary in a library in general.

Reply:
2. I'm asking about whether mismatched versions of `tracing` in `scylla` crate and the end crate can cause incompatility issues, resulting in misbehaving tracing.

Reply:
No idea, probably. It's a question to tracing crate maintainers, how are they going to handle new major version to avoid ecosystem split.